resources/i18n/EnchantmentCracker_pl.properties

```diff
@@ -0,0 +1,105 @@
+enchCrack.xpCost2.tooltip = Liczba po prawej stronie w drugim slocie
+enchCrack.xpCost3.tooltip = Liczba po prawej stronie w trzecim slocie
+enchCrack.check = Sprawdź
+enchCrack.check.tooltip = Użyj zebranych informacji aby zawęzić obszar poszukiwania ziarna XP
+enchCrack.reset = Reset
+enchCrack.reset.tooltip = Zresetuj ostatnie ziarno XP w celu złamania nowego ziarna
+enchCrack.impossible = Brak możliwch ziarn
+enchCrack.result = Ziarno XP: %08X
+enchCrack.remaining = Liczba możliwych ziarn: %s
+enchCrack.remaining.thousand = Liczba możliwych ziarn: %sk
+enchCrack.remaining.million = Liczba możliwych ziarn: %sM
+enchCrack.remaining.billion = Liczba możliwych ziarn: %sB
+enchCrack.progress = Postęp: %02.0f%%
+
+# Enchantment Calculator tab translations
+enchCalc.item.tooltip = Przedmiot który chcesz zakląć
+enchCalc.material.tooltip = Materiał przedmiotu którego chcesz zakląć
+enchCalc.maxBookshelves.tooltip = Maksymalna ilość półek na książki do użycia
+enchCalc.calculate = Oblicz
+enchCalc.calculate.tooltip = <html>Naciśnij by obliczyć jak wiele blocków/przedmiotów <i>musisz</i> wyrzucić żeby otrzymać upragnione zaklęcia</html>
+enchCalc.playerSeedNotFound = Musisz najpierw złamać ziarno gracza!\nPrzejdź do pierwszej zakładki i to zrób
+enchCalc.impossible = Niemożliwe
+enchCalc.noDummy = Bez kukły
+enchCalc.stackFormat = 64x%d + %d
+enchCalc.done = Dalej
+enchCalc.done.tooltip = Naciśnij aby dać znać programowi że wybrałeś zaklęcie ze stołu i przejść dalej
+enchCalc.throwCount.tooltip = Liczba bloków/przedmiotów które musisz wyrzucić (indywidualnie po sztuce)
+enchCalc.slot.tooltip = Slot który masz wybrać (1-3)
+enchCalc.bookshelves.tooltip = Liczba półek na książki które masz użyć
+enchCalc.level = Poz.
+enchCalc.level.tooltip = Aktualny poziom gracza. Pozwala na znajdowanie zaklęć na poziomie 30 i poniżej
+
+# About tab translation
+# End each line with "\n\". Links must be the whole line, and are in the format "LINK <url> <text>".
+# When translating into other languages, make sure to indicate what links are in English, and instruct the user to report bugs in English.
+program.about = Enchantment Cracker %s\n\
+  Wersja orginalna wykonana przez: Earthcomputer\n\
+  Optymalizacja i Udoskonalenie UI wykonane przez: Hexicube\n\
+  \n\
+  Poradnik i wyjaśnienia (Angielski):\n\
+  LINK https://youtu.be/hfiTZF0hlzw Minecraft, Vanilla Survival: Cracking the Enchantment Seed\n\
+  \n\
+  Imgur album (Angielski):\n\
+  LINK https://imgur.com/a/oaxCC5x MC enchantment Cracker Tutorial\n\
+  \n\
+  Strona Github(Angielski):\n\
+  LINK https://github.com/Earthcomputer/EnchantmentCracker Earthcomputer/EnchantmentCracker\n\
+  \n\
+  Prosimy o zgłaszanie błędów w zakładce issue tracker w języku angielskim i załączeniu pliku o nazwie \n\
+  enchcracker.log
+
+# Enchantments
+# These should be the same as the official translations of enchantments, abbreviated where too long
+ench.protection = Ochrona.
```

Comment:
Are these enchantments supposed to have periods (.) at the end?

Reply:
No. They are not necessary.

Reply:
Only "ench.silk_touch = Jedwabny dot." has shortened word at the end